docs/source/tutorials/multiannotator.ipynb

```diff
@@ -710,7 +710,7 @@
     "You can also repeatedly iterate this process of getting better consensus labels using the model's out-of-sample predicted probabilities and then retraining the model with the improved labels to get even better predicted probabilities!\n",
     "For details, see our [examples](https://github.com/cleanlab/examples) notebook on [Iterative use of Cleanlab to Improve Classification Models (and Consensus Labels) from Data Labeled by Multiple Annotators](https://github.com/cleanlab/examples/blob/master/multiannotator_cifar10/multiannotator_cifar10.ipynb).\n",
     "\n",
-    "If possible, the best way to improve your model is to collect additional labels for both previously annotated data and extra not-yet-labeled examples. To decide which data is most informative to label next, use `cleanlab.multiannotator.get_active_learning_scores()` rather than the methods shown here. This is demonstrated in our [examples](https://github.com/cleanlab/examples) notebook on [Active Learning with Multiple Data Annotators](https://github.com/cleanlab/examples/blob/master/active_learning_multiannotator/active_learning.ipynb).\n",
+    "If possible, the best way to improve your model is to collect additional labels for both previously annotated data and extra not-yet-labeled examples (i.e. *active learning*). To decide which data is most informative to label next, use `cleanlab.multiannotator.get_active_learning_scores()` rather than the methods from this tutorial. This is demonstrated in our [examples](https://github.com/cleanlab/examples) notebook on [Active Learning with Multiple Data Annotators via ActiveLab](https://github.com/cleanlab/examples/blob/master/active_learning_multiannotator/active_learning.ipynb).\n",
```

Comment:
committed your suggestion, but FYI that examples link was there for redundancy in case the latter notebook-specific link breaks (much more likely to happen than the link to the examples repo)